config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+class Rack::Attack
+  class Request < ::Rack::Request
+    # Take remote IP from Cloudfare's headers instead of rev proxy IP
+    def remote_ip
+      if ENV["CF_PROXY_DISABLED"]
```

Comment:
Assume CF is disabled by default

Reply:
If the env var is for some reason not set on the server, then that will result in all requests getting blocked. So I assumed enabled by default. Should I change it to disabled?

Reply:
It is easy for us to set this ENV, but for people who self host need documentation if are enabling this by default

Reply:
Okay I'll change it then :+1: